src/main/java/school/hei/haapi/service/utils/InstantUtils.java

```diff
@@ -9,7 +9,7 @@
 import school.hei.haapi.model.CourseSession;
 
 public class InstantUtils {
-  public static Instant Instant_now = LocalDateTime.now().atZone(ZoneId.of("UTC+3")).toInstant();
+  public static Instant now = LocalDateTime.now().atZone(ZoneId.of("UTC+3")).toInstant();
```

Comment:
this is not normal at all.

Reply:
there is a reason why Instant.now() is a method rather than an attribute, think about it x)